java/src/org/openqa/selenium/grid/node/local/LocalNode.java

```diff
@@ -806,19 +806,27 @@ private Session createExternalSession(
 
     // Check if the user wants to use BiDi
     boolean webSocketUrl = toUse.asMap().containsKey("webSocketUrl");
-    // Add se:bidi if necessary to send the bidi url back
-    boolean bidiSupported = isSupportingBiDi || toUse.getCapability("se:bidi") != null;
+    boolean bidiSupported = isSupportingBiDi || toUse.getCapability("webSocketUrl") != null;
     if (bidiSupported && bidiEnabled && webSocketUrl) {
-      String bidiPath = String.format("/session/%s/se/bidi", other.getId());
-      toUse = new PersistentCapabilities(toUse).setCapability("se:bidi", rewrite(bidiPath));
+      String biDiUrl = (String) other.getCapabilities().getCapability("webSocketUrl");
```

Comment:
Why don't we rewrite the URL and keep the endpoint? We could put the URL in `webSocketUrl`.

Reply:
https://github.com/SeleniumHQ/selenium/issues/13302#issuecomment-1918459493 The rationale behind the changes. Let me know what you think or if I am missing something. I did try the approach suggested earlier but that did not work due to not having the port information.

Reply:
I am sorry, I missed those comments. 

What if we store the "local" `webSocketUrl` in a different field and then retrieve that URL when the BiDi endpoint is hit? In the same way we do it with `debuggerAddress`.

As in `se:cdp` maps to `debuggerAddress`, we could map `webSocketUrl` to `se:gridWebSocketUrl`.

Just and idea.

Reply:
Sounds good!